eo-maven-plugin/src/main/java/org/eolang/maven/OyCaching.java

```diff
@@ -55,6 +55,9 @@ public final class OyCaching implements Objectionary {
      * @param ver Version.
      * @param cache Cache directory.
      * @param primary Primary objectionary.
+     * @todo #625:30m link changes in OyRemote with this class
```

Comment:
@yasamprom I'm not sure I understand this message. Need more explanation about the actions you expect someone after you to do

Reply:
Well, I think it is outdated todo and needs another discussion. I'll remove it

Reply:
Well, I think it is outdated todo and needs another discussion. I'll remove it